public/js/ChatController.js

```diff
@@ -1,6 +1,6 @@
-    /* global _, angular */
+    /* global _, angular, console */
```

Comment:
You shouldn't be using console in your actual code. I assume this is left over from debugging.